sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_constants.py

```diff
@@ -2,3 +2,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 VALID_NAME_CHARS = set("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789._-")
+
+FL_TESTING_COMPONENTS_FOLDER = "./tests/test_configs/components/fl_test_components"
+FL_TESTING_LOCAL_DATA_FOLDER = "./tests/test_configs/fl-e2e-test-data"
```

Comment:
This _really_ doesn't feel like the right spot for constants that seem _only_ needed for testing.

If you only need to use them in one file, consider moving them to that file.

If you need them across multiple test files, consider making a writing pytest fixtures in `conftest.py`

```python
def federated_learning_components_folder() -> Path:
    return Path("./tests/test_configs/components/fl_test_components")

def federated_learning_local_data_folder() -> Path:
    return Path("./tests/test_configs/fl-e2e-test-data")
```

Reply:
Move these to fixtures.